packages/preferences/src/browser/style/index.css

```diff
@@ -247,8 +255,6 @@
 }
 
 .theia-settings-container .single-pref:focus-within .pref-context-gutter .settings-context-menu-btn,
-.theia-settings-container .pref-name:hover+.pref-context-gutter .settings-context-menu-btn,
-.theia-settings-container .pref-context-gutter:hover .settings-context-menu-btn,
```

Comment:
With these changes (I think), it's now possible to hover over where the gear button should be and not reveal it if the preference line isn't focused - it remains opacity 0.

Reply:
Good catch, I noticed that these changes weren't needed at all :)